docs/templates/feature.md

```diff
@@ -2,42 +2,41 @@
 
 *Features* are the most granular entity in the [feature store](feature_store.md) and are
 logically grouped by [feature groups](feature_group.md).
+Features in the same feature groups are semantically related to the feature groups primary keys.
 
 The storage location of a single feature is determined by the
 [feature group](feature_group.md). Hence, enabling a [feature group](feature_group.md)
 for online storage will make a feature available as an online feature.
 
-New features can be appended to [feature groups](feature_group.md), however, to drop
-features, a new [feature group](feature_group.md) version has to be created. When
-appending features it is possible to specify a default value which is used for existing
-feature vectors in the [feature group](feature_group.md) for the new feature.
+## Features Taxonomy
 
-{{feature}}
+Within a [feature group](feature_group.md) there are different categories of features:
+
+* **Primary Keys**: These features describe the entity contained in a feature group (e.g. customer features, user features, ...). Within a feature group there could be more than one feature describing the primary key.
+* **Event Time**: One feature within a feature group can be selected to identify the time at which the event for the given record has happened. The event time feature is going to be used to enforce Point-in-time correctness when joining features from different feature groups together. When storing feature data on the offline feature store, each record is uniquely identified by the union of the primary key features and the even time feature.
+* **Partition Keys**: These features describe the storage layout of the feature group data on the offline feature store. Partition keys allow to divide the feature group data into different subdirectory. Partitioning can help to query more efficiently the data from the offline feature store.
+* **Normal Features**: These features represent attributes of related to the primary keys.
 
 ## Feature Types
```

Comment:
These are not feature types.
These are "Data Storage Types" in the online and offline stores.
A feature type is - categorical, numerical, embedding, etc.